packages/python/patch-no-srand

```diff
@@ -0,0 +1,23 @@
+--- Modules/expat/xmlparse.c
++++ Modules/expat/xmlparse.c
+@@ -20,7 +20,9 @@
+ #include <string.h>                     /* memset(), memcpy() */
+ #include <assert.h>
+ #include <limits.h>                     /* UINT_MAX */
++#if !defined(__CloudABI__)
```

Comment:
This chunk of the diff can be removed, I guess. There is no harm with including `<time.h>`.

We also had a similar patch in `packages/expat`, but the latest version of expat will require a different patch anyway. They changed the PRNG to use something equally bad.